share/transition_scripts/6.6.0/openldap2ds389.rb

```diff
@@ -0,0 +1,179 @@
+#!/opt/puppetlabs/puppet/bin/ruby
```

Comment:
It's a little hard to follow this logic―it uses `@instance` variables as if they were globals, without any classes.

If the logic was wrapped in a class, the instance variables might make more sense and we could spec test the script to demonstrate that the logic works.